src/sentry/api/endpoints/integrations/sentry_apps/components.py

```diff
@@ -55,9 +55,7 @@ def get(self, request: Request, organization) -> Response:
 
             for component in _components:
                 try:
-                    sentry_app_components.Preparer.run(
-                        component=component, install=install, project=project
```

Comment:
If you can believe it, the project was never used by the component anyways -- which also makes this easier for HC 😅